org.eclipse.dawnsci.analysis.api/src/org/eclipse/dawnsci/analysis/api/processing/IOperationBean.java

```diff
@@ -11,43 +11,147 @@
 
 import java.util.List;
 
+import com.sun.xml.internal.ws.api.streaming.XMLStreamReaderFactory.Default;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of Sun classes by ones from the Java API. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1191)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.sun.xml.internal.ws.api.streaming.XMLStreamReaderFactory.Default'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)